main/src/ca/uwaterloo/flix/tools/pkg/ManifestParser.scala

```diff
@@ -469,6 +487,31 @@ object ManifestParser {
     Ok(stringSet.toList)
   }
 
+  /**
+    * Creates the `IncludedModules` object from `optArray`.
+    */
+  private def toIncludedModules(optArray: Option[TomlArray], p: Path): Result[IncludedModules, ManifestError] = {
+    optArray match {
+      case None =>
+        Ok(IncludedModules.All)
+      case Some(array) =>
+        var moduleSet = Set.empty[Symbol.ModuleSym]
+        for (i <- 0 until array.size()) {
+          try {
+            val string = array.getString(i)
```

Comment:
What happens if the array contains a non-string? Maybe we should do this validation in the parsing part?